test/unit_tests/hmm.py

```diff
@@ -0,0 +1,116 @@
+from wiser.data.dataset_readers import MediaDatasetReader
```

Comment:
Perhaps better to call the folder "integration_tests" since these aren't really unit tests.

Also, fine to have separate scripts if needed to check different models, but Travis really should run the tutorials. Otherwise we won't know if users will hit any errors.